pom.xml

```diff
@@ -584,7 +584,7 @@
       <dependency>
         <groupId>org.apache.commons</groupId>
         <artifactId>commons-lang3</artifactId>
-        <version>3.3.2</version>
+        <version>3.5</version>
```

Comment:
I'm interested in `3.5` because Spark uses that: https://github.com/apache/spark/blob/master/pom.xml#L173 and we're using the neo4j frontend from our Spark-dependent project.